### DISCUSSION TITLE
Quick Tag: Catagories

### DISCUSSION THREAD

**Discussion by lyrieux** [AprilSylph/XKit-Rewritten#792] [Suggestion Box]
As someone with a lot of blogs and the wish to keep everything neat and tidy and functional without getting overwhelmed, the option to add a way to **categorize tags** would be amazing; just so they are not all mixed together but also to not have to scroll through irrelevant ones in order to get to the ones I want, per blog. Especially since I have so very many and I know I'm not alone in that fact-

**Comment by JRHB314** [comment]
Sounds interesting, but I'm not sure I understand. Categorize tags where? Like in the tag suggestion pop up? Something on your blog?

**Reply by marcustyphoon** [reply]
The title references Quick Tags, so I assume the post is talking about Quick Tags bundles.

**Reply by JRHB314** [reply]
Ah, that's pretty obvious huh? ;^^ Somehow I glanced over the "quick" in the title.

**Comment by lyrieux** [comment]
Omg, I'm sorry I'm atrocious at explaining anything. For me, for example, if I could choose a header like "Aesthetics" as some kind of drop down and then beneath it is all the tag bundles relevant to it. So I can hide the ones I don't need at that moment in time and save having to scroll through an entirely huge list to find the one I want, :) 

If that... makes any more sense. Might be easier if I draw it, aha-!

**Reply by hellersqueen** [reply]
if someone can do that, honestly thatd be so great

**Comment by PrincessPrince** [comment]
Yes!! I'd love this I have a ton of blogs and a ton of quick tags and it'd be nice to have some quick tags only show up when that blog is selected from the drop down. It'd be very useful especially bc it'd declutter the quick tag menu in general and i'd add more than I have already!!
